code/ATMOSPHERICS/atmospherics.dm

```diff
@@ -44,6 +44,8 @@ GLOBAL_DATUM_INIT(pipe_icon_manager, /datum/pipe_icon_manager, new())
 	if(!pipe_color_check(pipe_color))
 		pipe_color = null
 
+	update_all_pipe_vision()
```

Comment:
Why in the New and not the Initialize?

Reply:
Not really sure. Is there a standard? I need this to happen each time a new pipe is constructed.